### DISCUSSION TITLE
User accounts - Only require for settings/admin

### DISCUSSION THREAD

**Discussion by harrybadger** [bpatrik/pigallery2#490] [Q&A]
Hi,

Firstly, I'd like to say this project is fantastic!

Is it possible to only require login for the admin/settings page?
I'd like it so anybody can view the gallery/albums, but a user login is required to change the settings.
I had a go at trying to do this myself by creating a fork but I'm very new to this and got stuck!

I've tried removing authentication lines from router.ts files but haven't managed to get anywhere.

Any advice would be appreciated.

Edit:
I've had a bit more of a play, and it looks like sharing the gallery without a password gets me half the way there, since it creates a gallery that you don't need a user account to view.
Is it possible to set this gallery link as the default path for the server instead of going to login by default?

**Comment by bpatrik** [comment]
you can try to just disable password at all in the config.json. You will be still able to change settings in the config.json itself but every one be able to see the photos.
